### PR TITLE
chore(package): downgrade http dep

### PR DESCRIPTION
**Pull request by polsieira-wk**
## Ultimate problem:
wdesk has a hard constraint of `http = ^0.12.2` making json_schema 3.1 incompatible with wdesk

## How it was fixed:
Downgrade `http`

## Potential areas of regression:

---

> __FYA:__ @michaelcarter-wf

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -9,16 +9,15 @@ environment:
 dependencies:
   args: '>=0.13.7 <3.0.0'
   collection: ^1.14.6
-  http: ^0.13.3
+  http: ">=0.9.0 <0.13.0"
```
Comment (michaelcarter-wf) [inline pubspec.yaml:12]: Do we want to limit this to 0.12 just to save us headaches when we upgrade again soonish?

**Comment by aviary3-wk** [comment]
## Security Insights
The items listed below may not capture all security relevant changes. Before providing a security review, be sure to review the entire PR for security impact.
<details>
<summary>(4) Security relevant changes were detected</summary>
<li>Watched keyword <code>Access-Control-Allow-Origin</code> in <code>tool/cors_headers.dart</code> line(s) ['3'] added</li>
<li>Watched keyword <code>Access-Control-Allow-Headers</code> in <code>tool/cors_headers.dart</code> line(s) ['6'] added</li>
<li>Watched keyword <code>Access-Control-Allow-Methods</code> in <code>tool/cors_headers.dart</code> line(s) ['7'] added</li>
<li>Watched keyword <code>Access-Control-Max-Age</code> in <code>tool/cors_headers.dart</code> line(s) ['8'] added</li>
</details>

## Action Items
* Obtain a [security review](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); reviewer should pay special attention to insights listed above
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Review by michaelcarter-wf — CHANGES_REQUESTED** [review]

**Review by michaelcarter-wf — DISMISSED** [review]
+1

**Review by michaelcarter-wf — APPROVED** [review]
+1 to Pol's changes

**Review by michaelcarter-wf — APPROVED** [review]
+1

**Review by robbielamb-wf — APPROVED** [review]

**Comment by michaelcarter-wf** [comment]
+10

- [x] CI Passes
- [x] compatible with http 0.12.x

**Comment by michaelcarter-wf** [comment]
@Workiva/release-management-p

**Comment by michaelcarter-wf** [comment]
@Workiva/release-management-p

**Review by bryanrezende-wf — APPROVED** [review]
Security +1
- Changes to cors_headers look safe
- No other concerning changes

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**